src/utils/robust.spec.ts

```diff
@@ -1,71 +1,58 @@
-import { convertError, reportError } from 'utils/report-error';
-import { _, catchErrorsWithContext } from './robust';
+import { reportError } from './report-error';
+import { catchErrorsAndReport } from './robust';
 import type { Modules } from './robust';
 
-const { catchAndLogError } = _;
-
-jest.mock('utils/report-error', () => ({
-	convertError: jest.fn((e: Error) => e),
+jest.mock('./report-error', () => ({
```

Comment:
Refactored this module to improve mocking and test single function now exported from `robust`